packages/components/nodes/tools/ChatflowTool/ChatflowTool.ts

```diff
@@ -242,7 +260,7 @@ class ChatflowTool extends StructuredTool {
             question: inputQuestion,
             chatId: flowConfig?.chatId,
```

Comment:
might need to replace `chatId` with a new uuid as well

Reply:
That's actually a question I had. What's the difference between ChatID and SessionID? Should both be the same?

Reply:
From what I tried, replacing just the SessionID is enough to make it work, but I also saw that ChatID is currently the same as SessionID when the tool is called.

Reply:
yeah for example, when using OpenAI Assistant:
sessionID = thread_abc (this is an ID from OpenAI)
chatId = uuid (this is an ID from Flowise)

in other words, chatId is sessionID specifically for flowise, whereeas sessionID is from 3rd party

Reply:
Done, I pushed a change generating also a random id for the chatId